### PR TITLE
Moves kubectl get subcommand to staging

### PR DESCRIPTION
**Pull request by seans3**
* Moves `kubectl get` subcommand to staging
* Updates BUILD files to accommodate move

/kind cleanup
/sig cli
/area kubectl
/priority important-soon

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/get/BUILD [hunk 1]
```diff
@@ -72,11 +73,7 @@ go_test(
         "humanreadable_flags_test.go",
         "sorter_test.go",
     ],
-    data = [
-        "//api/openapi-spec",
-        "//test/e2e/testing-manifests:all-srcs",
-        "//test/fixtures",
-    ],
+    data = ["//staging/src/k8s.io/kubectl/test/data"],
```
Comment (liggitt) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/BUILD]: preexisting, but I'm slightly surprised this wasn't under `test/data`. Any reason not to rename the folder in a follow-up?

**Comment by liggitt** [comment]
/lgtm
/approve

finally! :)

File: staging/src/k8s.io/kubectl/pkg/cmd/get/BUILD (hunk 1 above)
Comment (seans3) [reply staging/src/k8s.io/kubectl/pkg/cmd/get/BUILD]: No. I'll change it in the next PR. This was my fault in a previous PR.

**Pull request merged**